log/level.go

```diff
@@ -8,6 +10,12 @@ const defaultLogLevelKey = "*"
 //
 // Example:
 // ParseLogLevel("consensus:debug,mempool:debug,*:error", "info")
+// 
+// This function attemps to keep the same behavior as the CometBFT ParseLogLevel
 func ParseLogLevel(lvl string, defaultLogLevelValue string) (func(key, level string) bool, error) {
```

Comment:
Aims to keep functionality parity with https://github.com/cometbft/cometbft/blob/v0.34.27/libs/cli/flags/log_level.go#L22